rllib/BUILD

```diff
@@ -1343,13 +1343,6 @@ py_test(
 #
 # Tag: evaluation
 # --------------------------------------------------------------------
-py_test(
```

Comment:
Moved the important test therein into `test_env_runner_failures`.
All other tests therein are redundant and are scenarios are already covered in `test_env_runner_failures`!